### ISSUE TITLE
spot bug in SGDW implementation (weight decay part)

### ISSUE THREAD

**Issue by Leiay** [jettify/pytorch-optimizer#454]
Hi,

I was using the SGDW implementation in this repo, and I wonder if anything is wrong with this line:

https://github.com/jettify/pytorch-optimizer/blob/910b414565427f0a66e20040475e7e4385e066a5/torch_optimizer/sgdw.py#L121

Let weight decay be $\lambda$ and learning rate be $\alpha$. If I understand it correctly, this line of code update weight decay with
$$\theta_t \leftarrow \theta_t - \lambda \alpha$$

where it should be
$$\theta_t \leftarrow \theta_t \cdot (1 - \lambda \alpha)$$

This result in poor performance of training compared to SGD with the same set of optimization hyper-parameter.

Thanks!

Regards, Liu
